doc/man3/BIO_meth_new.pod

```diff
@@ -118,7 +118,9 @@ for creating a new instance of the BIO respectively. This function will be
 called in response to the application calling BIO_new() and passing
 in a pointer to the current BIO_METHOD. The BIO_new() function will allocate the
 memory for the new BIO, and a pointer to this newly allocated structure will
-be passed as a parameter to the function.
+be passed as a parameter to the function. If a create function is set,
+the create function must call L<BIO_set_init(3)> to mark the BIO as initialised
```

Comment:
Please note that the BIO_set_init might not be called by the create function itself but perhaps even later by some ctrl() function. Could you please amend the documentation slightly to reflect that?

Reply:
@t8m done.